bigquery-firestore-export/functions/package.json

```diff
@@ -29,7 +29,7 @@
     "typescript": "^5.8.2"
   },
   "devDependencies": {
-    "@google-cloud/pubsub": "^4.10.0",
+    "@google-cloud/pubsub": "^5.1.0",
```

Comment:
the bump to `@google-cloud/pubsub` from `4.10.0` to `5.1.0` is a major and results in failing tests. will be reverting this

Reply:
yes please - only collect up the minor dependency bumps in these PRs